application/models/User.php

```diff
@@ -910,7 +939,7 @@ public function getColums()
 
         $cols[] = array(
             "name" => "parentUserName",
-            "header" => gT("Created by"),
+            "header" => gT("Owner"),
```

Comment:
Owner of a user ?

Reply:
Good question: What happens if the creator of a user is deleted.

Reply:
I know, but owner of user : seems strange, near owner of people